### DISCUSSION TITLE
Trying to add env variables on the Vercel site for a project - form is blocked?

### DISCUSSION THREAD

**Discussion by drscottlobo** [vercel/community#3229] [Bug Report]
### Page to Investigate

/my-project/settings/environment_variables

### Steps to Reproduce

I went to try to add ENV variables to a new project on the Vercel site today, and the form for adding env variables is blocked - the form seems like it is set to disabled -, there is no way to enter any info in any of the fields. Then I check another project that is up and running fine, but same issue, no way to add environment variables?

**Comment by zchsh** [comment]
I've run into this same issue as well.  The form for adding new variables is disabled, with some kind of `.geist-disabled` scrim overtop of it with `cursor: not-allowed`.

I found I could set the environment variables using `vercel env add` through the [vercel CLI](https://vercel.com/docs/cli/env). From there I can even then edit existing variables through the Vercel UI.

This seems like probably something accidental on Vercel's part, but not sure?

**Reply by drscottlobo** [reply]
Yep, that's exactly my experience. Thanks for the suggestion! I'll try the CLI as a work-around. 

Yeah, definitely seems like some attribute on the form is just broken on the website.

**Comment by salmon-coder** [comment]
OK glad I'm not going crazy here!

Tried to mess around with the html to enable it but couldn't figure it out so I guess I'll develop locally until they fix it :man_shrugging:

**Reply by drscottlobo** [reply]
CLI worked just fine!

**Comment by dmitriy-MI6** [comment]
The same thing now.
Bypassed it by deleting all 'disabled' props from inputs via DevTools

**Reply by lesliewubbel** [reply]
How do you do that? 'disabled' props from inputs via DevTools

**Reply by dmitriy-MI6** [reply]
@Gresliebear remove disabled props from input
move <input disabled /> to <input />, the same thing about textarea

**Reply by lesliewubbel** [reply]
deleting this?
![image](https://github.com/vercel/community/assets/13157730/2c86e9aa-48e2-437a-ba3a-223ad4b3a6b6)

**Reply by estevaz12** [reply]
When you inspect the key `input` element and the value `textarea`, remove the `disabled` attribute.

**Comment by amyegan** [comment]
Thanks for reporting this! We're looking into it. 🙇 

As others have mentioned, you can use the [CLI](https://vercel.com/docs/cli/env#) as a temporary workaround.

**Reply by lesliewubbel** [reply]
Thank you

**Reply by amyegan** [reply]
A bugfix was released and I can confirm that problem has gone away for me. Please let me know if you're still seeing this issue 🙏

**Reply by estevaz12** [reply]
@amyegan Yup, it's working now! Thanks!

**Reply by drscottlobo** [reply]
Yep, all better! Thanks for the quick response!
